metadata/metadata.go

```diff
@@ -171,6 +172,15 @@ func (m *MetadataClient) getContainersDnsRecords(dnsEntries map[string]utils.Met
 	return nil
 }
 
+func (m *MetadataClient) updateEnvironmentName() {
+	envName, _, err := getEnvironment(m.MetadataClient)
+	if err != nil {
+		logrus.Fatalf("Error updating env info: %v", err)
```

Comment:
I'd return an error instead of fataling, and bubble up the error in the caller method

Reply:
done. thanks for your review